app/serializers/event_serializer.rb

```diff
@@ -1,4 +1,6 @@
 class EventSerializer < ActiveModel::Serializer
   attributes :id, :name, :description, :image, :url, :date, :location, :instructions, :contact, :campaign_id
 
+  belongs_to :campaign
+  has_many :tasks
```

Comment:
will this work without a task serializer? we probably should have one anyway, right?